code/scripting/lua.cpp

```diff
@@ -4637,6 +4637,40 @@ ADE_FUNC(isBeam, l_Weaponclass, NULL, "Return true if the weapon is a beam", "bo
 		return ADE_RETURN_FALSE;
 }
 
+// Checks if a weapon has been paged in (counted as used)
+ADE_FUNC(isWeaponUsed, l_Weaponclass, NULL, "Return true is the weapon is paged in, optionally paging in the weapon.", "boolean", "true if the weapon is paged in, pages in the weapon if otherwise")
+{
+	int idx;
```

Comment:
Leaving `idx` uninitialized is probably not a good idea (we've had plenty of problems with uninitialized variables in the Lua interface already).